data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,366 @@
+			"blast radius" 100
+			"shield damage" 1000
+			"hull damage" 500
+			"hit force" 1500
+	outfits
+		`"Biroo" Atomic Steering`
+		`"Biroo" Atomic Thruster`
+		"Boulder Reactor"
+		"Bullfrog Anti-Missile" 2
+		"Cargo Expansion" 2
+		"Hai Diamond Regenerator"
+		"Hai Fissure Batteries"
+		"Hai Williwaw Cooling"
+		Hyperdrive
+		"Ion Cannon" 3
+		"Quantum Keystone"
+
+	gun 0 -69.5 "Ion Cannon"
+	gun -12 -68.5 "Ion Cannon"
+	gun 12 -68.5 "Ion Cannon"
+	turret -36 3 "Bullfrog Anti-Missile"
+	turret 36 3 "Bullfrog Anti-Missile"
+	engine -36 62 0.8
+	engine 36 62 0.8
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "This sensor-laden exploration vessel hails back to a more peaceful era of the Hai. Though they have no true ramscoop, a technology the Hai never developed, their built-in collection gills magnify the natural ramscoop present in all ships. These expensive features are the reason the ship has been replaced with the Lightning Bug."
+	description "This ship is now appropriate as an agile, long distance cargo hauler and blockade runner. It is able to defend itself in small skirmishes, even though you suspect that it does not operate at its full potential, as if it stayed underwater for some time... The simple fact that it is still running at all after so many years is clearly proof of the durability of Hai technology."
+
+mission "Hai Legacy 1"
+	invisible
+	landing
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		not "event: hai ladybug available"
+	on offer
+		conversation
+			`As you enter the shipyard you quickly get approached by someone wearing a hood, apparently not too keen to reveal their identity. They come to address you, discreetly. Seeing you stop to listen, they take your arm and make you follow them. "Keep walking," they say, quickly taking a look around the way someone looking for a new ship would, before addressing you again. "My associates and I have a business proposition for you, if you are interested to learn more about it go to these coordinates." They give you a piece of paper containing the location, a beach almost on the other side of the planet with an instruction to not go with your ship, before quickly fading into the crowd. Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			`	You proceed to hire a motorized antigrav vehicle to go there, and arrive around sunset. You set foot after double checking you are indeed at the right coordinates, for except a cave, vegetation, sand, and, of course, the beach, you cannot see much of interest in the vicinity in the dimming light. You can't help but keep your hand close to your gun, feeling safer with its presence.`
```

Comment:
I'll second Tibe's comments on the writing, but also here; if this is on Hai-home it'd make much more sense that you'd get on a train or something. You almost certainly don't have a license to operate a Hai vehicle according to Hai road rules.

If it's problematic for you to be seen asking how to buy a ticket, then maybe they should just give you one.

Reply:
Good point

Reply:
```suggestion
			`	As you ponder how to make your way to the rendez-vous point, you notice a conveniant train ticket in your pocket, with instructions on how to find your way to the destination. You proceed to take it and arrive around sunset without any complications, even though it took a lot of walking to get there. You look around you, double checking you are indeed at the right coordinates, for except a cave, vegetation, sand, and, of course, the beach, you cannot see much of interest in the vicinity in the dimming light. You can't help but keep your hand close to your gun, feeling safer with its presence.`
```